packages/store/addon/-private/system/deprecated-evented.js

```diff
@@ -0,0 +1,72 @@
+import Evented from '@ember/object/evented';
+import Mixin from '@ember/object/mixin';
+import { deprecate } from '@ember/debug';
+import { DEBUG } from '@glimmer/env';
+
+let INSTANCE_DEPRECATIONS;
+let lookupDeprecations;
+let DeprecatedEvented;
+
+if (DEBUG) {
+  INSTANCE_DEPRECATIONS = new WeakMap();
+
+  lookupDeprecations = function lookupInstanceDrecations(instance) {
+    let deprecations = INSTANCE_DEPRECATIONS.get(instance);
+
+    if (!deprecations) {
+      deprecations = {};
+      INSTANCE_DEPRECATIONS.set(instance, deprecations);
+    }
+
+    return deprecations;
+  };
+
+  DeprecatedEvented = Mixin.create(Evented, {
+    _has(name) {
+      return Evented.mixins[0].properties.has.call(this, name);
+    },
+
+    deprecrateEvented(eventName) {
```

Comment:
the methodName here is misspelled (`deprecate` not `deprecrate`). Later we also attempt to invoke this as `deprecatedEvented()`. We should probably make it `_deprecateEvent(eventName)` as the signature.